pulpcore/cli/common/context.py

```diff
@@ -689,7 +744,7 @@ def scope(self) -> Dict[str, Any]:
         return {self.repository_ctx.HREF: self.repository_ctx.pulp_href}
 
     def repair(self, href: Optional[str]) -> Any:
-        return self.call("repair", parameters={self.HREF: href or self.pulp_href})
+        return self.call("repair", parameters={self.HREF: href or self.pulp_href}, body={})
```

Comment:
Why does this require an empty dict for `body`? Can it not be left out?

Reply:
It requires a body with only an optional property.

`pulp debug openapi operation --id repositories_rpm_rpm_versions_repair | jq '.operation.requestBody'`
```json
{
  "content": {
    "application/json": {
      "schema": {
        "$ref": "#/components/schemas/Repair"
      }
    },
    "application/x-www-form-urlencoded": {
      "schema": {
        "$ref": "#/components/schemas/Repair"
      }
    },
    "multipart/form-data": {
      "schema": {
        "$ref": "#/components/schemas/Repair"
      }
    }
  },
  "required": true
}
```
`pulp debug openapi schema --name Repair`
```
{
  "type": "object",
  "properties": {
    "verify_checksums": {
      "type": "boolean",
      "default": true,
      "description": "Will verify that the checksum of all stored files matches what saved in the database. Otherwise only the existence of the files will be checked. Enabled by default"
    }
  }
}
```

But I could add a check for the `required` attribute in the `responseBody`.

Reply:
Accounted for the `responseBody.required` now. This lead to a new subroutine `render_request_body`.